python/ray/ml/preprocessors/scaler.py

```diff
@@ -0,0 +1,92 @@
+from typing import List
+
+import pandas as pd
+
+from ray.data import Dataset
+from ray.data.aggregate import Mean, Std, Min, Max
+from ray.ml.preprocessor import Preprocessor
+
+
+class StandardScaler(Preprocessor):
+    """Scale values within columns based on mean and standard deviation.
+
+    For each column, each value will be transformed to ``(value-mean)/std``,
+    where ``mean`` and ``std` are calculated from the fitted dataset.
+
+    Args:
+        columns: The columns that will individually be scaled.
+    """
+
+    def __init__(self, columns: List[str]):
+        super().__init__()
+        self.columns = columns
+
+    def _fit(self, dataset: Dataset) -> Preprocessor:
+        # TODO: confirm what to set ddof as for calculating Std.
```

Comment:
sklearn uses a biased estimator for the standard deviation, i.e. `ddof=0`. Meanwhile, our `Std` aggregation defaults to 1 to match Pandas behavior. If we're wanting to match sklearn behavior, we should use `Std(ddof=0)`.

Reply:
Ah, interesting! I set the default behavior to use `ddof=0` and exposed a `ddof` parameter to the class so the user can override.

Probably not a huge deal either way:
> We use a biased estimator for the standard deviation, equivalent to numpy.std(x, ddof=0). Note that the choice of ddof is unlikely to affect model performance.